dev-requirements.txt

```diff
@@ -5,3 +5,4 @@ mock==2.0.0 ; python_version == '2.7'
 pytest-cov>=2.4.0,==2.*
 vcrpy>=1.10.0,<=1.10.9
 vcrpy-unittest==0.1.6
+joblib~=0.11
```

Comment:
If the container has to have the civis client for the joblib backend stuff and the civis client depends on joblib, then I don't think we need to install joblib on the fly above, right?